test/models/concerns/audit_extensions_test.rb

```diff
@@ -46,4 +46,13 @@ def setup
     FactoryGirl.create(:ec2_cr, :with_auditing)
     refute_empty Audit.search_for("type = compute_resource")
   end
+
+  test "audit change field can be greater then 65K bytes" do
+    prov_template = templates(:mystring)
+    prov_template.template = "0" * 66000
```

Comment:
Nitpick: I think this might be faster a bit:

     prov_template.template = "0000000000" * 6600

```
[root@zzzap katello-3.4.0.rc2]# time ruby -e '"0000000000" * 99999999'

real    0m4.588s
user    0m0.256s
sys     0m0.124s
[root@zzzap katello-3.4.0.rc2]# time ruby -e '"0" * 999999990'

real    0m0.352s
user    0m0.240s
sys     0m0.112s
```

Even faster would be String.new("", capacity: 66000) and appending it in a loop perhaps.